nirum_wsgi.py

```diff
@@ -44,6 +45,17 @@
 ])
 
 
+def _get_argument_value(payload, key, type_):
+    # if is_union_type(type_) is true, type_ is optional type.
+    # https://github.com/spoqa/nirum-python/blob/maintenance-0.6/nirum/deserialize.py#L209
+    if key in payload or is_union_type(type_):
```

Comment:
We'd better have `nirum._compat.is_optional_type()` or such thing which tests whether it's a union type and one of its possible types is `type(None)`.